docs/how_to/apphooks.rst

```diff
@@ -308,9 +309,7 @@ will look like this::
 
     class OscarApp(CMSApp):
         name = _("Oscar")
-        urls = [
-            patterns('', *application.urls[0])
```

Comment:
In sample app in tests for this case `urls` parameter is fixed already. So, i think this is not updated docs about hooks. If tests is passed then it will be ok.

Sample app:
https://github.com/vovanbo/django-cms/blob/master/cms/test_utils/project/sampleapp/cms_apps.py#L16-L22